app/controllers/clients/show/dois/new.js

```diff
@@ -23,6 +23,8 @@ export default Controller.extend({
         doi.set('descriptions', [{ description: doi.get('descriptions'), descriptionType: 'Abstract' }]);
       }
 
+      doi.set("source", "fabricaForm");
```

Comment:
@richardhallett you can also do this directly when you create the DOI, i.e. start with defaults.